packages/truffle-core/lib/commands/migrate.js

```diff
@@ -134,7 +134,7 @@ const command = {
     const Resolver = require("truffle-resolver");
     const Migrate = require("truffle-migrate");
     const Config = require("truffle-config");
-    const Environment = require("../environment");
+    const Environment = require("truffle-environment").environment;
```

Comment:
```suggestion
    const { Environment } = require("truffle-environment");
```

Reply:
love this! will make the change! actually meant to ask you about a prettier way to do this. :)